### PR TITLE
fix  [DEV] kernel parameters 'console=tty0 console=ttyS0' are not passed to the installed Ubuntu 16.04 LTS node #731@github

### PR DESCRIPTION
**Pull request by immarvin**
enable bootup information display on console during ubuntu 16.04 reboot

### CODE REVIEW

File: xCAT-server/share/xcat/install/scripts/post.xcat [hunk 1]
```diff
@@ -379,6 +379,11 @@ if [[ $OSVER == ubuntu* ]]; then
     elif [[ -r /boot/grub/grub.cfg ]] ; then
         update-grub
     fi
+
+    [ -f /etc/default/grub ] && sed -i -e '/GRUB_CMDLINE_LINUX_DEFAULT=/s/quiet//;s/splash//' -e '/#GRUB_TERMINAL=/s/#//' -e 's/#GRUB_GFXMODE=.*/GRUB_GFXMODE=text/' /etc/default/grub && update-grub
+    if echo "$ARCH" | grep -i 'x86_64'; then
+        cat /boot/grub/grub.cfg |grep -e "console=tty0 *console=ttyS0" >/dev/null 2>&1 || ([ -f /etc/default/grub ] && sed -i -e '/GRUB_CMDLINE_LINUX_DEFAULT=/s/="/=" console=tty0 console=ttyS0 /' /etc/default/grub && update-grub)
```
Comment (daniceexi) [inline xCAT-server/share/xcat/install/scripts/post.xcat]: we should not use the fixed port ``ttyS0``. We may take the console cfg from /pro/cmdline.

**Comment by immarvin** [comment]
hi @daniceexi , code updated

**Comment by immarvin** [comment]
verified on ubuntu 16.04 x86_64 kvm guest

**Comment by daniceexi** [comment]
I have reviewed the change and agree to merge.

**Pull request merged**